backend/src/modules/product/product.schema.ts

```diff
@@ -0,0 +1,21 @@
+import { Schema } from 'mongoose';
+
+export const ProductSchema = new Schema({
+  name: String,
+  description: String,
+  category: String,
+  brand: String,
+  price: Number,
+  sizes: [String],
+  colors: [String],
+  images: [String],
+  sliderImage: String,
+});
+
+ProductSchema.set('toJSON', {
+  transform: function (doc, ret, options) {
+      ret.id = ret._id;
```

Comment:
It would be better to use 
https://stackoverflow.com/questions/22229609/exclude-fields-from-result-in-mongodb-monk

Reply:
referenced in issue